indico/modules/events/management/forms.py

```diff
@@ -232,6 +232,11 @@ def __init__(self, *args, **kwargs):
             del self.label
             del self.label_message
 
+    def validate_keywords(self, field):
+        allowed_keywords = set(misc_settings.get('allowed_keywords'))
+        if allowed_keywords and field.data and not all(kw in allowed_keywords for kw in field.data):
```

Comment:
use a set operation

Reply:
`allowed_keywords` is already a set.

Reply:
I meant replacing `not all(kw in allowed_keywords for kw in field.data)` with a set operation

```python
allowed_keywords = set(...)
keywords = set(field.data)
if allowed_keywords and not (keywords <= allowed_keywords):
    raise ...
```

Reply:
Done!